### PR TITLE
Fixed links

### PR DESCRIPTION
**Pull request by LorenzNickel**
Both links can not be accessed any longer, for one I found an alternative

### CODE REVIEW

**Comment by LorenzNickel** [comment]
Okay, can somebody help me to pass this check? Seems like the new url in my snapshot does not match something else, how can I fix this?

File: packages/relay-compiler/core/GraphQLSchemaUtils.js [hunk 1]
```diff
@@ -82,8 +82,6 @@ function canHaveSelections(type: GraphQLType): boolean {
  * Implements duck typing that checks whether a type has an id field of the ID
  * type. This is approximating what we can hopefully do with the __id proposal
  * a bit more cleanly.
- *
- * https://github.com/graphql/graphql-future/blob/master/01%20-%20__id.md
```
Comment (kassens) [inline packages/relay-compiler/core/GraphQLSchemaUtils.js]: This link still exist. As long as we have to comment above, it makes sense for me to keep?

Comment (LorenzNickel) [reply packages/relay-compiler/core/GraphQLSchemaUtils.js]: I don't really understand what you mean. If I open this link (or just [the repo of it](https://github.com/graphql/graphql-future/)) I get a 404-Github page. Is this repo private?
image

Comment (kassens) [reply packages/relay-compiler/core/GraphQLSchemaUtils.js]: Oh, I guess it's been made private as the last change was 3 years ago.

**Pull request closed without merging**